test/mwcas_descriptors_test.cpp

```diff
@@ -88,15 +88,17 @@ class MwCASDescriptorFixture : public ::testing::Test
       target_fields_[i] = 0UL;
     }
 
-    if constexpr (std::is_same_v<MwCASDesc, AOPT> || std::is_same_v<MwCASDesc, CASN>) {
+    if constexpr (std::is_same_v<MwCASDesc, AOPT> || std::is_same_v<MwCASDesc, CASN>
+                  || std::is_same_v<MwCASDesc, LFMwCAS>) {
       MwCASDesc::StartGC();
     }
   }
 
   void
   TearDown() override
   {
-    if constexpr (std::is_same_v<MwCASDesc, AOPT> || std::is_same_v<MwCASDesc, CASN>) {
+    if constexpr (std::is_same_v<MwCASDesc, AOPT> || std::is_same_v<MwCASDesc, CASN>
+                  || std::is_same_v<MwCASDesc, CASN>) {
```

Comment:
修正漏れですね。プロセス終了時に解放されるので実は害はないのですが。

```suggestion
                  || std::is_same_v<MwCASDesc, LFMwCAS>) {
```